bodhi/server/util.py

```diff
@@ -297,9 +298,23 @@ def hostname(context=None):
 
 
 def markup(context, text):
-    return markdown.markdown(text, safe_mode="replace",
-                             html_replacement_text="--RAW HTML NOT ALLOWED--",
-                             extensions=['markdown.extensions.fenced_code'])
+    """
+    Return HTML from a markdown string
+
+    Args:
+        context:
+        text (basestring): markdown text to be converted to HTML
+
+    Return:
+        basestring: HTML representation of the markdown text
+    """
+
+    # previously, we used the Safe Mode in python-markdown to strip all HTML
+    # tags. Safe Mode is deprecated, so we now use Bleach to sanitize all HTML
+    # tags before running it through the markdown parser
+    sanitized_text = bleach.clean(text, tags=[])
```

Comment:
Looks like python-bleach-1.4.1-2.el7 is in epel7, which is what bodhi runs on.  (Confirmed, that's exactly the version available to the `bodhi03` VM).

(Someone could always put the latest bleach > 2.0 in the epel7-infra tag).